its/ruling/src/test/resources/expected/kotlin/kotlin-language-server/kotlin-S1192.json

```diff
@@ -0,0 +1,8 @@
+{
+'kotlin-kotlin-language-server-project:server/src/main/kotlin/org/javacs/kt/KotlinTextDocumentService.kt':[
+107,
```

Comment:
I created [SONARKT-248](https://sonarsource.atlassian.net/browse/SONARKT-248) to fix this false-positive.